build.sc

```diff
@@ -143,6 +143,42 @@ object generator extends SwamModule with PublishModule {
 
 }
 
+
+
+object interpreter extends SwamModule with PublishModule {
```

Comment:
Can you name it `cli`? I imagine we could have more than an interpreter in CLI.

Reply:
Also, I am not sure this should be published to maven repo since it is not a library. Can you remove the `PublishModule` mixin? I will make sure the CLI is uploaded on releasing.